docs/pages/order-by-accessors.rst

```diff
@@ -45,3 +45,75 @@ the ``orderable`` argument::
     class SimpleTable(tables.Table):
         name = tables.Column()
         actions = tables.Column(orderable=False)
+
+
+.. _table.order_foo:
+
+:meth:`table.order_FOO` methods
+--------------------------------
+
+Another solution for alternative ordering is being able to chain functions on to
+the original queryset. This method allows more complex functionality giving the
+ability to use all of Django's QuerySet API.
+
+Adding a `Table.order_FOO` method (where `FOO` is the name of the column),
+gives you the ability to chain to, or modify, the original queryset when that
+column is selected to be ordered.
+
+The method takes two arguments: `queryset`, and `is_descending`. The return
+must be a tuple of two elements. The first being the queryset and the second
+being a boolean; note that modified queryset will only be used if the boolean is
+`True`.
+
+For example, let's say instead of ordering alphabetically, ordering by
+amount of characters in the first_name is desired.
+The implementation would look like this:
+::
+
+    # tables.py
+    from django.db.models.functions import Length
+
+    class PersonTable(tables.Table):
+        name = tables.Column()
+
+        def order_name(self, queryset, is_descending):
+            queryset = queryset.annotate(
+                length=Length('first_name')
+            ).order_by(('-' if is_descending else '') + 'length')
+            return (queryset, True)
+
+
+
+As another example, presume the situation calls for being able to order by a
+mathematical expression. In this predicament, the table needs to be able to be
```

Comment:
I had to look up what predicament means, can we rephrase this to use a more common word?